agir/front/components/dashboardComponents/Footer.js

```diff
@@ -283,8 +284,8 @@ export const Footer = (props) => {
           <div>
             <h3>Action populaire</h3>
             <p>
-              {!isSignedIn && routes.login && (
-                <Link href={routes.login}>Se connecter</Link>
+              {!isSignedIn && (
+                <Link href={routeConfig.login.getLink()}>Se connecter</Link>
```

Comment:
Tu peux utiliser Link aussi directement avec la clé de la route :
```suggestion
                <Link route="login">Se connecter</Link>
```